src/generators/metadata/utils/parse.mjs

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+import { u as createTree } from 'unist-builder';
+import { findAfter } from 'unist-util-find-after';
+import { remove } from 'unist-util-remove';
+import { selectAll } from 'unist-util-select';
+import { SKIP, visit } from 'unist-util-visit';
+
+import createQueries from '../../../utils/queries/index.mjs';
+import { getRemark } from '../../../utils/remark.mjs';
+import { createNodeSlugger } from '../../../utils/slugger/index.mjs';
+import createMetadata from '../../../metadata.mjs';
```

Comment:
nit: are any of these only in this generator? If so, can we move them here?

Reply:
I moved the slugger file, but `metadata.mjs` types is also used on `src/utils/queries/index.mjs`

Reply:
No problem, just checking :-)